yt_dlp/extractor/zdf.py

```diff
@@ -248,19 +258,48 @@ class ZDFIE(ZDFBaseIE):
             'timestamp': 1675160100,
             'upload_date': '20230131',
             'thumbnail': 'https://epg-image.zdf.de/fotobase-webdelivery/images/e2d7e55a-09f0-424e-ac73-6cac4dd65f35?layout=2400x1350',
+            'series': 'SOKO Stuttgart',
+            'seriesId': 'f862ce9a-6dd1-4388-a698-22b36ac4c9e9',
+            'season': 'Staffel 11',
+            'season_number': 11,
+            'season_id': 'ae1b4990-6d87-4970-a571-caccf1ba2879',
+            'episode': 'Das Geld anderer Leute',
+            'episode_number': 10,
+            'episode_id': 'POS_7f367934-f2f0-45cb-9081-736781ff2d23',
         },
     }, {
         'url': 'https://www.zdf.de/dokumentation/terra-x/unser-gruener-planet-wuesten-doku-100.html',
         'info_dict': {
-            'id': '220605_dk_gruener_planet_wuesten_tex',
+            'id': '220525_green_planet_makingof_1_tropen_tex',
             'ext': 'mp4',
-            'title': 'Unser grüner Planet - Wüsten',
-            'description': 'md5:4fc647b6f9c3796eea66f4a0baea2862',
-            'duration': 2613.0,
-            'timestamp': 1654450200,
-            'upload_date': '20220605',
-            'format_note': 'uhd, main',
-            'thumbnail': 'https://www.zdf.de/assets/saguaro-kakteen-102~3840x2160?cb=1655910690796',
+            'title': 'Making-of Unser grüner Planet - Tropen',
+            'description': 'md5:d7c6949dc7c75c73c4ad51c785fb0b79',
+            'duration': 435.0,
+            'timestamp': 1653811200,
+            'upload_date': '20220529',
+            'format_note': 'hd, main',
+            'thumbnail': 'https://www.zdf.de/assets/unser-gruener-planet-making-of-1-tropen-100~3840x2160?cb=1653493335577',
+            'episode': 'Making-of Unser grüner Planet - Tropen',
```

Comment:
somehow this video changed completely. it also no longer matches what is visible on the webpage. But it also says "Video no longer available" on the page, so that might explain the discrepancy. Still let me verify that it matches the player2 behaviour.

![image](https://github.com/yt-dlp/yt-dlp/assets/36509607/d6689216-323c-4965-8138-8ddde7ee198b)

Reply:
I switched back to master to check:

```
python devscripts/run_tests.py ZDFIE_7 
Running ['pytest', '-Werror', '--tb=short', 'test/test_download.py::TestDownload::test_ZDF_7']

...

E   AssertionError: '220605_dk_gruener_planet_wuesten_tex' != '220525_green_planet_makingof_1_tropen_tex'
E   - 220605_dk_gruener_planet_wuesten_tex
E   + 220525_green_planet_makingof_1_tropen_tex
E    : Invalid value for field id, expected '220605_dk_gruener_planet_wuesten_tex', got '220525_green_planet_makingof_1_tropen_tex'
```

So it matches previous behaviour, we should be good here.